ui/app/AppLayouts/Onboarding/views/InsertDetailsView.qml

```diff
@@ -151,7 +151,9 @@ Item {
                 onKeyPressed: {
                     if (input.edit.keyEvent === Qt.Key_Return || input.edit.keyEvent === Qt.Key_Enter) {
                         event.accepted = true
-                        nextBtn.clicked(null)
+                        if(nextBtn.enabled) {
```

Comment:
since it seems this is the only action here, wouldn't it make sense to move this check in the upper statement so that if the button is not enabled to prevent getting into here?